src/lib/generators/code-snippet-generators.spec.ts

```diff
@@ -24,8 +101,24 @@ describe('Generators', () => {
 
     const expectedSVGConstant = `export const ${variableName}: ${interfaceName} = {
                 name: '${filenameWithoutEnding}',
-                data: '${data}'
+                data: \`${data}\`
```

Comment:
\` should not be needed, ${data} should be enough

Reply:
I'm comparing the strings, and the output does have backticks, without it the spec fails.

Reply:
Okay - makes sense